.eas/build/send-slack-message.yml

```diff
@@ -0,0 +1,34 @@
+build:
+  name: Slack your team when build completes
+  steps:
+    # For the purpose of the example-test we don't need to do the actual build.
+    # - eas/build
+
+    - eas/send_slack_message:
+        name: Send Slack message to a given webhook URL
+        inputs:
+          message: 'This is a test message to plain input URL'
+          slack_hook_url: 'https://hooks.slack.com/services/[rest_of_hook_url]'
```

Comment:
Actually… will this fail the job (Slack responding with 400 or something) or will we let the rest of the workflow pass? If it fails maybe we want to comment it out not to mark the job as failed (nor skip the rest of the workflow)?

Reply:
I don't see why would that be a problem - it's an example workflow, and the url obviously needs to be updated with the actual value. Unless I'm missing something?

Reply:
Ah, sure, [I was thinking](https://github.com/expo/eas-custom-builds-example/pull/15#pullrequestreview-1872499717) we could enable a Github trigger for this workflow and get messages in #turtle-bots every time we commit to `main` here. We can not do it too.

Reply:
Ah, gotcha. I'll look into that